journeys/et/dockerCommon.ts

```diff
@@ -47,6 +47,10 @@ async function getDefaultTasks(tasks: TASK_CHOICES) {
 }
 
 export async function configsJourney() {
+  if ((await execCommand('docker info', undefined, false)).stdout.includes("The command 'docker' could not be found")) {
```

Comment:
```suggestion
 const { stdout } = await execCommand('docker info', undefined, false)
 if (stdout.includes("The command 'docker' could not be found")) {
```

Would prefer splitting out into two lines please. Also, are you sure it comes from stdout and not stderr? I'm aware docker has a dodgy track record of printing progress to stderr and errors to stdout, but have you confirmed this works?